### PR TITLE
Removed duplicate entries

### LINKED ISSUE #142: Duplicate section title
screen shot 2016-10-05 at 12 48 05 am

See: https://github.com/WordPress/twentyseventeen/blob/master/style.css#L582-L588

### PR DESCRIPTION
**Pull request by maheshwaghmare**
Fixed #142

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -3366,4 +3362,4 @@ article.panel-placeholder {
 	.twentyseventeen-front-page .entry-content blockquote.alignright {
 		margin-right: -20%;
 	}
-}
\ No newline at end of file
+}
```
Comment (josephfusco) [inline style.css:3365]: This is an unrelated change

File: style.css [hunk 2]
```diff
@@ -583,10 +583,6 @@ q:after {
 # Forms
 --------------------------------------------------------------*/
 
-/*--------------------------------------------------------------
-# Forms
---------------------------------------------------------------*/
-
```
Comment (melchoyce) [inline style.css:589]: This was fixed in https://github.com/WordPress/twentyseventeen/commit/033c71099840c0462c7f48c2addcbbfffc00ebf2
